docs/kratos/manage-identities/25_import-user-accounts-identities.mdx

```diff
@@ -293,14 +293,20 @@ Ory Identities doesn't hash passwords by salted SHA but can compare stored salte
 {SSHA}JFZFs0oHzxbMwkSJmYVeI8MnTDy/276a
 ```
 
+20 bytes of hash followed by the salt
```

Comment:
I guess it can be a bit confusing if you only talk about bytes here. Because we use base64 encoding, one character (typically represented as one byte in ASCII/UTF-8) has only 6 bits of information.
```suggestion
The format consists of 20 bytes of hash followed by the salt. The result is then encoded with base64 following RFC 4648.
```